arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -46,7 +40,13 @@
             "schema": {
               "$ref": "#/definitions/WebService"
             }
-          }                
+          },                
```

Comment:
What you had was correct. ResourceGroupName should be the first parameter then comes Rp specific parameter name and in the end should be the body parameter. That is the general convention. If this was not being followed previously then that was incorrect.

Check this example: https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-01-01/swagger/storage.json#L64. subscriptionId and api-version will be client properties in most cases hence their order is not that important.